src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -59,6 +61,30 @@ export const mhvBaseUrl = () => {
   return `https://${mhvSubdomain}.myhealth.va.gov`;
 };
 
+const mhvToEauthRoutes = {
+  'download-my-data': 'download_my_data',
+  'web/myhealthevet/refill-prescriptions': 'prescription_refill',
+  'secure-messaging': 'secure_messaging',
+  appointments: 'appointments',
+  'labs-tests': 'lab-tests',
+};
+
+const mhvUrlNonSSO = subroute => {
+  const mhvSubdomain = !environment.isProduction() ? 'mhv-syst' : 'www';
```

Comment:
small thing, but is there a reason for negating the condition?

Reply:
Ahh, good catch. That's part of the existing code, but I agree that negated flags are more confusing to read. I'll refactor it to the reverse